packages/node/package.json

```diff
@@ -48,14 +52,10 @@
     "@types/node-fetch": "^2.6.1",
     "@types/test-listen": "1.1.0",
     "@vercel/build-utils": "4.1.1-canary.1",
-    "@vercel/ncc": "0.24.0",
```

Comment:
I noticed the version here doesn't match

Reply:
I think we should stick with `"@vercel/ncc": "0.24.0"` for now since that was the version before #7925